packages/preview-middleware-client/src/adp/quick-actions/table-quick-action-base.ts

```diff
@@ -239,17 +261,16 @@ export abstract class TableQuickActionDefinitionBase {
      * Processes table element and pushes table data to the children array
      * @param table - table element
      * @param sectionInfo - section info object
+     * @param onAddChild - Optional callback executed when a child element is added to update tooltip adn enable property.
```

Comment:
There is no 3rd parameter in this function.

Reply:
removed.